upfadapter/pfcp/udp/pfcp_message_type.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2024 Canonical Ltd.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package udp
+
+import "github.com/wmnsk/go-pfcp/message"
+
+func IsRequest(msg message.Message) bool {
+	switch msg.MessageType() {
+	case message.MsgTypeHeartbeatRequest:
+		return true
+	case message.MsgTypePFDManagementRequest:
+		return true
+	case message.MsgTypeAssociationSetupRequest:
+		return true
+	case message.MsgTypeAssociationUpdateRequest:
+		return true
+	case message.MsgTypeAssociationReleaseRequest:
+		return true
+	case message.MsgTypeNodeReportRequest:
+		return true
+	case message.MsgTypeSessionSetDeletionRequest:
+		return true
+	case message.MsgTypeSessionEstablishmentRequest:
+		return true
+	case message.MsgTypeSessionModificationRequest:
+		return true
+	case message.MsgTypeSessionDeletionRequest:
+		return true
+	case message.MsgTypeSessionReportRequest:
+		return true
+	default:
+		return false
+	}
+}
+
+func IsResponse(msg message.Message) bool {
+	switch msg.MessageType() {
+	case message.MsgTypeHeartbeatResponse:
+		return true
+	case message.MsgTypePFDManagementResponse:
+		return true
+	case message.MsgTypeAssociationSetupResponse:
+		return true
+	case message.MsgTypeAssociationUpdateResponse:
+		return true
+	case message.MsgTypeAssociationReleaseResponse:
+		return true
+	case message.MsgTypeNodeReportResponse:
+		return true
+	case message.MsgTypeSessionSetDeletionResponse:
+		return true
+	case message.MsgTypeSessionEstablishmentResponse:
+		return true
+	case message.MsgTypeSessionModificationResponse:
+		return true
+	case message.MsgTypeSessionDeletionResponse:
+		return true
+	case message.MsgTypeSessionReportResponse:
+		return true
```

Comment:
What about something like this? or you can change it to an `if` statement
```suggestion
	case message.MsgTypeHeartbeatResponse, message.MsgTypePFDManagementResponse, message.MsgTypeAssociationSetupResponse, message.MsgTypeAssociationUpdateResponse, message.MsgTypeAssociationReleaseResponse message.MsgTypeNodeReportResponse message.MsgTypeSessionSetDeletionResponse message.MsgTypeSessionEstablishmentResponse message.MsgTypeSessionModificationResponse message.MsgTypeSessionDeletionResponse, message.MsgTypeSessionReportResponse:
		return true
```

Reply:
Done